CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## Version 4.4.0 (February 17, 2017)
+
+Heuristic changes:
+
+- ftfy can now fix mojibake involving the Windows-1250 or ISO-8859-2 encodings.
+
+- The `fix_entities` fixer is now applied after `fix_encoding`. This makes
+  more situations resolvable when both fixes are needed.
```

Comment:
The main `fix_text` function iterates until it reaches a fixed point, so this should still work. The problem was that I saw a real case where decoding HTML first created a character that wasn't in the encoding, so the encoding fixer got stuck.